frontend/src/app/components/theme.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { BrowserRouter as Router, Route } from 'react-router-dom'
+import * as ROUTES from '../const/routes'
+import { Header } from './header'
+import Footer from './Footer/footer'
+import { Container } from '../../styles/blocks/layout'
+import { RootActions } from '../modules/root/root-actions'
+// pages
+import { Dashboard } from '../pages/Dashboard'
+import { CreateMatchPage } from '../pages/CreateMatch'
+import { Profile } from '../pages/Profile'
+import { AddUserPage } from '../pages/AddUser'
+import { MatchListPage } from '../pages/MatchList'
+import { UserListPage } from '../pages/UserList'
+
+const ThemeComponent = ({theme, themeTransition, changeTheme, stopThemeTransition}) => {
+  const transition = themeTransition === 1 ? " theme-transition" : ""
```

Comment:
i ran the app on localhost and the linter seemed to be screaming all the time. in this instance about the double quotes. Do you have a linter installed?

Reply:
no, I have no linter installed. Are we talking about IDE extension I suppose? React linter pops up a lot of hits, which one shall I choose?

Reply:
@petrk-salsita exmplained to me, I would create another mini PR to add lint into scripts in `package.json` file and to resolve any issues that might pop out form it.

Reply:
[done](https://github.com/salsita/foosball-rating/pull/81/commits/93d14d9d4dfbc131f4c797ffe52478fa61f2614f) and [created](#90 )